docs/source/contribution.rst

```diff
@@ -431,15 +431,22 @@ To build the documentation, you need to install `Sphinx <http://www.sphinx-doc.o
 
   $ pip install sphinx sphinx_rtd_theme
 
-Then you can build the documentation in HTML format locally::
+Docstrings (documentation comments in the source code) are collected from the installed Chainer module.
+If Chainer has not been installed yet, then ``make html`` will yield an error that the reference sources cannot
+be read, such as ``chainer.AbstractSerializer``.
```

Comment:
>such as ``chainer.AbstractSerializer``.

What does this mean?

Reply:
It's a function I often saw an error on. Edited to remove specific reference.